src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -15,4 +17,9 @@ export default interface ClientMetricReport {
    * Gets video client media metrics
    */
   getObservableVideoMetrics?(): { [id: string]: { [id: string]: {} } };
+
+  /**
+   * Gets raw RTCStatsReport from `RTCPeerConnection.getStats()` API
```

Comment:
I don't think here is the right place for those. I will add some explanation in the migration doc.

Reply:
>if IDEs will auto-populate

Yes

>it would be natural for developers to check on the API documentation when they are actively coding

Make sense, I will add some brief instruction there.

Reply:
Ack. We probably want to update our metrics guide after @LeviSklaroff finishes the av bandwidth stuff. This should go in there also.